LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2021 Dataiku
+   Copyright 2023 Dataiku
```

Comment:
Actually this should be a range 2021-2023
It has to have the oldest date in there